src/services/ApiClient.ts

```diff
@@ -135,12 +135,18 @@ class ApiClient {
       .then(res => res.data)
   }
 
-  async postSimpleSearchQuery(request: SearchQuery) {
+  async postSimpleSearchQuery(request: SearchQuery) {    
     return await client
       .post(ROUTES.POST_SIMPLE_SEARCH, request)
       .then(res => res.data)
   }
 
+  async getLinkedRecord(request: SearchQuery) {
+    return await client
+      .post(ROUTES.POST_SEARCH_REQUEST, request)
+      .then(res => res.data)
+  }
+
```

Comment:
We are getting the linked records, why do we need this ?

Reply:
Sure